public/app/features/alerting/unified/components/rule-list/AlertRuleListItem.tsx

```diff
@@ -0,0 +1,384 @@
+  health?: RuleHealth;
+  isProvisioned?: boolean;
+  lastEvaluation?: string;
+  evaluationInterval?: string;
+  evaluationDuration?: number;
+  labels?: Labels;
+  instancesCount?: number;
+}
+
+export const AlertRuleListItem = (props: AlertRuleListItemProps) => {
+  const {
+    name,
+    summary,
+    state,
+    health,
+    error,
+    href,
+    isProvisioned,
+    lastEvaluation,
+    evaluationDuration,
+    evaluationInterval,
+    isPaused = false,
+    instancesCount = 0,
+    labels,
+  } = props;
+  const styles = useStyles2(getStyles);
+
+  return (
+    <li className={styles.alertListItemContainer} role="treeitem" aria-selected="false">
+      <Stack direction="row" alignItems="start" gap={1} wrap="nowrap">
+        <RuleListIcon state={state} health={health} isPaused={isPaused} />
+        <Stack direction="column" gap={0.5} flex="1">
+          <div>
+            <Stack direction="column" gap={0}>
+              <Stack direction="row" alignItems="start">
+                <TextLink href={href} inline={false}>
+                  {name}
+                </TextLink>
+                {labels && <AlertLabels labels={labels} size="xs" />}
+              </Stack>
+              <Summary content={summary} error={error} />
+            </Stack>
+          </div>
+          <div>
+            <Stack direction="row" gap={1}>
+              {/* if the rule is paused we don't care about the evaluation metadata */}
+              {isPaused ? (
+                <></>
+              ) : (
+                <EvaluationMetadata
+                  lastEvaluation={lastEvaluation}
+                  evaluationDuration={evaluationDuration}
+                  evaluationInterval={evaluationInterval}
+                  health={health}
+                  error={error}
+                />
+              )}
+              {!isPaused && (
+                <>
+                  <MetaText icon="layer-group">
+                    <TextLink variant="bodySmall" color="secondary" href={href + '?tab=instances'} inline={false}>
+                      {pluralize('instance', instancesCount, true)}
+                    </TextLink>
+                  </MetaText>
+
+                  {!isEmpty(labels) && (
+                    <MetaText icon="tag-alt">
+                      <TextLink variant="bodySmall" color="secondary" href={href} inline={false}>
+                        {pluralize('label', labelsSize(labels), true)}
+                      </TextLink>
+                    </MetaText>
+                  )}
+                </>
+              )}
+            </Stack>
+          </div>
+        </Stack>
+
+        <Stack direction="row" alignItems="center" gap={1} wrap="nowrap">
+          <Button
+            variant="secondary"
+            size="sm"
+            icon="pen"
+            type="button"
+            disabled={isProvisioned}
+            aria-label="edit-rule-action"
+            data-testid="edit-rule-action"
+          >
+            Edit
```

Comment:
Use an i18n string according to https://github.com/grafana/grafana/blob/main/contribute/internationalization.md here

Reply:
👍 I'll create a separate follow-up PR to add locale support to alerting